README.md

```diff
@@ -7,20 +7,57 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/simple-icons-font@v2/font/simple-icons.min.css"
```

Comment:
I think we should add the example for both JSDelivr and Unpkg, like in the main repo, agreed?

Reply:
Also, wow this works I actually wasn't sure it would find the font files just like that :sweat_smile:

Reply:
> I think we should add the example for both JSDelivr and Unpkg, like in the main repo, agreed?

Yes!

> Also, wow this works I actually wasn't sure it would find the font files just like that sweat_smile

CSS magic :sparkles: It seems that relative calls in `url()` are relative to the URL of file calling them, which it's very nice.